apps/mail/lib/hotkeys/use-hotkey-utils.ts

```
@@ -52,11 +52,63 @@ const isMac =
+  "'": '-',
+  ',': 'w',
+  '.': 'v',
+  '/': 'z',
+  '-': '[',
+  '[': '/',
+  ']': '=',
+  '=': ']',
+};
+
+const qwertyToDvorak: Record<string, string> = Object.entries(dvorakToQwerty).reduce(
+  (acc, [dvorak, qwerty]) => ({ ...acc, [qwerty]: dvorak }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)